### PR TITLE
Rename to ClaimableOutputs

### LINKED ISSUE #572: Rename get_unlockable_outputs_with_additional_unlock_conditions
To something UX-friendly

### PR DESCRIPTION
**Pull request by thibault-martinez**
Fixes https://github.com/iotaledger/iota-sdk/issues/572

### CODE REVIEW

File: sdk/src/wallet/account/operations/balance.rs [hunk 1]
```diff
@@ -165,16 +165,14 @@ where
 
                                 let account_addresses = self.addresses().await?;
                                 let local_time = self.client().get_time_checked().await?;
-                                let output_can_be_unlocked_now = self
-                                    .get_unlockable_outputs_with_additional_unlock_conditions(OutputsToClaim::All)
-                                    .await?
-                                    .contains(output_id);
+                                let claimable_outputs =
```
Comment (Alex6323) [inline sdk/src/wallet/account/operations/balance.rs]:
```suggestion
                                let is_claimable =
```

File: sdk/src/wallet/account/operations/balance.rs [hunk 2]
```diff
@@ -165,16 +165,14 @@ where
...
+                                    self.claimable_outputs(OutputsToClaim::All).await?.contains(output_id);
 
                                 // For outputs that are expired or have a timelock unlock condition, but no expiration
                                 // unlock condition and we then can unlock them, then
                                 // they can never be not available for us anymore
                                 // and should be added to the balance
-                                if output_can_be_unlocked_now {
+                                if claimable_outputs {
```
Comment (Alex6323) [inline sdk/src/wallet/account/operations/balance.rs]:
```suggestion
                                if is_claimable {
```

File: sdk/src/wallet/account/operations/output_claiming.rs [hunk 3]
```diff
@@ -411,7 +408,11 @@ pub(crate) fn sdr_not_expired(output: &Output, current_time: u32) -> Option<&Sto
                 .map_or(false, |expiration| current_time >= expiration.timestamp());
 
             // We only have to send the storage deposit return back if the output is not expired
-            if !expired { Some(sdr) } else { None }
+            if !expired {
+                Some(sdr)
+            } else {
+                None
+            }
```
Comment (Alex6323) [inline sdk/src/wallet/account/operations/output_claiming.rs]:
```suggestion
            !expired.then_some(sdr)
```

Comment (thibault-martinez) [reply sdk/src/wallet/account/operations/output_claiming.rs]: You probably meant `(!expired).then_some(sdr)` but ok

File: sdk/src/wallet/account/operations/output_claiming.rs [hunk 4]
```diff
@@ -408,11 +408,7 @@ pub(crate) fn sdr_not_expired(output: &Output, current_time: u32) -> Option<&Sto
                 .map_or(false, |expiration| current_time >= expiration.timestamp());
 
             // We only have to send the storage deposit return back if the output is not expired
-            if !expired {
-                Some(sdr)
-            } else {
-                None
-            }
+            (!expired).then_some(sdr)
```
Comment (Alex6323) [inline sdk/src/wallet/account/operations/output_claiming.rs:411]: ah, forgot the parentheses :see_no_evil:

**Review by Alex6323 — DISMISSED** [review]

**Comment by qrayven** [comment]
does it require a new file in  `.changes` ?

**Review by qrayven — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Pull request merged**